segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceBuilder.java

```diff
@@ -66,13 +66,15 @@
 public class ServiceBuilder implements AutoCloseable {
     //region Members
 
-    private static final Duration SHUTDOWN_TIMEOUT = Duration.ofSeconds(30);
+    private static final Duration SHUTDOWN_TIMEOUT = Duration.ofSeconds(45);
```

Comment:
Why?

Reply:
This is to give some allowance as we will also be closing the new `lowPriorityExecutor` in addition to the two preexisting executors `coreExecutor` and `storageExecutor`. I just made a simplistic assumption here that if we give two executors `30` seconds, then we should give three of them `45`.